code/game/atoms_movable.dm

```diff
@@ -196,6 +196,15 @@
 	if(pulledby && moving_diagonally != FIRST_DIAG_STEP && get_dist(src, pulledby) > 1)		//separated from our puller and not in the middle of a diagonal move.
 		pulledby.stop_pulling()
 
+/atom/movable/proc/set_glide_size(target = 8)
+	glide_size = target
+
+	for(var/atom/movable/AM in contents)

```

Comment:
why are you recursing into contents.

most items that are in contents aren't visually viewable on the turf

Reply:
good point, this was how it was done in eris but they might use contents for buckled as well.